public/assets/js/index.js

```diff
@@ -0,0 +1,314 @@
+const signupForm = $(".sign-up-form");
+const loginForm = $(".log-in-form");
+const logOutBtn = $("#logout-btn");
+const createBlog = $(".create-blog");
+const editBlog = $(".edit-blog");
+const deleteBlog = $(".delete-btn");
+const commentForm = $(".comment-form");
+const deleteAComment = $(".delete-comment");
+
+const handleSignUp = async (event) => {
+  event.preventDefault();
+
+  const first_name = $("#first-name").val();
+  const last_name = $("#last-name").val();
+  const email = $("#email").val();
+  const username = $("#username").val();
+  const password = $("#password").val();
+  const confirmPassword = $("#confirm-password").val();
+  const alertMessage = $("#alert-message");
+
+  if (
+    first_name &&
+    last_name &&
+    email &&
+    username &&
+    password &&
+    confirmPassword
+  ) {
+    if (password === confirmPassword && password.length > 8) {
```

Comment:
This if statement and the if statement below mean that if a user inputs a password that is exactly 8 characterslong, they do not meet any of  the declarations so nothing happens. There are no error messages displayed and they cannot sign up. Amend this to say `>= 8` which would allow a password to be 8 characters long.